src/components/primitives/Input/index.tsx

```diff
@@ -192,7 +197,7 @@ const Input = (
                 transform: [{ translateY: slideAnim, translateX: 4 }],
               }}
             >
-              <Flex {...newProps} bg="transparent">
+              <Flex {...newProps} nativeID={undefined} bg="transparent">
```

Comment:
@Ajamuar this is a bit hack which I have discussed with @MD-REHMAN , we'll need to refactor Input. 
We'll have to select which props should be passed into Flex and which into Input.